cluster/gce/util.sh

```diff
@@ -783,6 +783,35 @@ function teardown-monitoring {
   fi
 }
 
+function setup-fluentd-elasticsearch-logging {
+  echo .. setting up Fluentd to Elasticsearch logging with Kibana viewer
+  local -r kubectl="${KUBE_ROOT}/cluster/kubectl.sh"
+  if "${kubectl}" create -f "${KUBE_ROOT}"/cluster/addons/fluentd-elasticsearch/es-pod.yml &> /dev/null && \
```

Comment:
We should decide on .yaml or .yml or .json and do the same thing everywhere.  DNS says .yaml, monitoring says .json, and you say .yml :)